tests/rptest/services/redpanda_cloud.py

```diff
@@ -907,14 +911,22 @@ def create(self, superuser: SaslCredentials) -> str:
             # Just create new cluster
             self._create_new_cluster()
 
+        # If network type is private, trigget VPC Peering
+        if not self.isPublicNetwork:
+            # Checks for if such VPC exists is not needed
+
+            # AWS will not create duplicate peering connection
+            #  and will just return existing one.
+
+            # GCP will fall to error if such peering exists
```

Comment:
Not quite following here: if GCP errors if a peering already exists, don't we need the check if such VPC exists?

Reply:
In case of AWS - it just returns the existing one.
In case of GCP - we do nothing for BYOC, it creates them by default. In case of FMC, we do not check for existing ones either, just creating them right away